### PR TITLE
Update swift-syntax links to swiftlang/swift-syntax

### PR DESCRIPTION
**Pull request by ahoppen**

### CODE REVIEW

**Comment by ahoppen** [comment]
@swift-ci Please test

File: Package@swift-6.0.swift
Comment (dennisweissmann) [inline Package@swift-6.0.swift:1]: Did you accidentally bring that back?

**Review by stmontgomery — CHANGES_REQUESTED** [review]
Thanks for fixing these references!

As @dennisweissmann mentioned, could you either explain why the Package@swift-6.0 file is being restored, or else remove it if that was accidental?

File: Package@swift-6.0.swift
Comment (ahoppen) [reply Package@swift-6.0.swift:1]: Oh yes. It must have re-appeared during a rebase.

**Comment by ahoppen** [comment]
@swift-ci Please test

**Review by stmontgomery — APPROVED** [review]

**Review by dennisweissmann — APPROVED** [review]

**Pull request merged**